packages/terra-application-menu-layout/tests/wdio/application-menu-spec.js

```diff
@@ -0,0 +1,15 @@
+/* global browser, Terra, before */
+
+const viewports = Terra.viewports('tiny', 'medium');
```

Comment:
@dkasper-was-taken @tbiethman Does this need tested for a any particular viewport size? Previously tests with large viewport